### ISSUE TITLE
ZeroScans' Second Life Ranker not showing all available chapters

### ISSUE THREAD

**Issue by Izunyaaan** [tachiyomiorg/tachiyomi-extensions#10132]
### Source information

ZeroScans 1.2.2

### Steps to reproduce

1. Select ZeroScans extension.
2. Search for Second Life Ranker.

### Expected behavior

Shows up to current chapter available on website (105 as of today)

### Actual behavior

Shows only up to chapter 68.

### Tachiyomi version

0.12.3

### Android version

Android 10

### Other details

I do hope I didnt miss another ticket already reporting this issue...

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by Seishirou101** [comment]
Considering that their website is at 108 and not 105, and its working fine with me. Are you sure you did troubleshooting?
![second life ranker](https://user-images.githubusercontent.com/57241064/146483238-ded131fa-dd29-4a24-a449-df3aa0bfd7c8.png)
![Screenshot_20211216-211033297](https://user-images.githubusercontent.com/57241064/146483262-4dc3f431-59c1-48f5-8e8f-e3523b558208.jpg)

**Comment by Izunyaaan** [comment]
Strange... And yes 108 not 105 that was a typo.
I've checked webview and thats how I discovered the issue in the first
place so its not cloudflare. And there's no error so not sure which
troubleshooting step to follow but most I have cleared cache cookies and
stuff already. Other than trying another internet connection I'm not sure
what to do.
Attached is screenshot of me opening the manhwa in tachiyomi.
